reference/docs-conceptual/community/contributing/powershell-style-guide.md

```diff
@@ -231,9 +188,9 @@ The following image file types are supported: `*.png`, `*.gif`, `*.jpeg`, `*.jpg
 ### Markdown extensions supported by Open Publishing
 
 The [Microsoft Docs Authoring Pack](/contribute/how-to-write-docs-auth-pack) contains tools that
```

Comment:
Microsoft Docs Authoring Pack link generates a 404.

Reply:
Same issue because of preview.